src/Castle.Windsor/Castle.Windsor.csproj

```diff
@@ -1,754 +1,41 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\buildscripts\common.props"></Import>
```

Comment:
The import line needs to be after the target frameworks - see original csproj file. The reason is that precompile symbols (defined in common.props) depend on the target framework. You can verify that by opening the solution in Visual Studio and look for any symbol that should be defined on net45, such as `#if FEATURE_REMOTING` and you'll see that the active/inactive code looks wrong. This is also why the tests fail.